amy/fiscal/tests/test_membership.py

```diff
@@ -894,95 +909,272 @@ def setUpMembership(self):
             membership=self.membership,
             role=MemberRole.objects.first(),
         )
+        MembershipTask.objects.create(
+            person=self.hermione,
+            membership=self.membership,
+            role=MembershipPersonRole.objects.first(),
+        )
 
     def test_form_simple_valid(self):
-        data = {
-            "name": "Test Name",
-            "consortium": True,
-            "public_status": "public",
-            "variant": "partner",
-            "agreement_start": "2021-03-01",
-            "agreement_end": "2022-03-01",
-            "contribution_type": "financial",
-            "workshops_without_admin_fee_per_agreement": 10,
-            "public_instructor_training_seats": 12,
-            "additional_public_instructor_training_seats": 0,
-            "inhouse_instructor_training_seats": 9,
-            "additional_inhouse_instructor_training_seats": 0,
-        }
-
-        form = MembershipRollOverForm(data)
-
+        form = MembershipRollOverForm(self.data)
         self.assertTrue(form.is_valid())
 
-    def test_form_ignoring_fields(self):
-        data = {
-            # field accepted
-            "name": "Test Name",
-            "consortium": True,
-            "public_status": "public",
-            "variant": "partner",
-            "agreement_start": "2021-03-01",
-            "agreement_end": "2022-03-01",
-            "contribution_type": "financial",
-            "workshops_without_admin_fee_per_agreement": 10,
-            "public_instructor_training_seats": 12,
-            "additional_public_instructor_training_seats": 0,
-            "inhouse_instructor_training_seats": 9,
-            "additional_inhouse_instructor_training_seats": 0,
-            # fields ignored
-            "workshops_without_admin_fee_rolled_from_previous": "invalid value",
-            "public_instructor_training_seats_rolled_from_previous": "invalid value",
-            "inhouse_instructor_training_seats_rolled_from_previous": "invalid value",
-        }
-
-        form = MembershipRollOverForm(data)
+    def test_form_validation_rolled_fields(self):
+        test_data = [
+            # PASS: centrally org. workshops maxed out, others set to 0
```

Comment:
I'm not sure I know what `centrally org` means

Reply:
_Centrally-organised workshops_ ie. workshops that are handled by The Carpentries staff. Our regional coordinators for example help with finding instructors and helpers.
_Self-organised workshops_ are administered entirely by instructors or other staff.